src/McBopomofo.cpp

```diff
@@ -741,6 +741,20 @@ void McBopomofoEngine::handleMarkingState(fcitx::InputContext* context,
 
 fcitx::CandidateLayoutHint McBopomofoEngine::getCandidateLayoutHint() const {
   fcitx::CandidateLayoutHint layoutHint = fcitx::CandidateLayoutHint::NotSet;
+
+  auto choosingCandidate =
+      dynamic_cast<InputStates::ChoosingCandidate*>(state_.get());
+  if (choosingCandidate != nullptr) {
+    auto candidates = choosingCandidate->candidates;
+    for (auto candidate : candidates) {
+      std::wstring_convert<std::codecvt_utf8<char32_t>, char32_t> conv;
```

Comment:
You can use `McBopomofo::CodePointCount(const std::string&)` (in `UTF8Helper.h`) for the same thing, so that we don't have to use the deprecated `codecvt`.

Reply:
Okay. Will do.